packages/video-ad-sdk/src/adContainer/helpers/__tests__/createIframe.spec.js

```diff
@@ -1,11 +1,17 @@
 import createIframe from '../createIframe';
+import supportsSrcdoc from '../supportsSrcdoc';
 
+jest.mock('../supportsSrcdoc');
+// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Can this be removed now? It doesn't look like the test is disabled.

Reply:
👍